packages/aws-cdk-lib/aws-eks/lib/alb-controller.ts

```diff
@@ -241,6 +241,27 @@ export interface AlbControllerOptions {
    * @default - Corresponds to the predefined version.
    */
   readonly policy?: any;
+
+  /**
+   * The values to be used by the chart.
+   * For nested values use a nested dictionary. For example:
+   * values: {
+   *  autoscaling: false,
+   *  ingressClassParams: { create: true }
+   * }
+   *
+   * Note that the following values are set by the controller and cannot be overridden:
+   *  - clusterName
+   *  - serviceAccount.create
+   *  - serviceAccount.name
+   *  - region
+   *  - vpcId
+   *  - image.repository
+   *  - image.tag
+   *
+   * @default - No values are provided to the chart.
+   */
+  readonly values?: {[key: string]: any};
```

Comment:
```suggestion
  readonly helmChartValues?: {[key: string]: any};
```

Can you rename this to `helmChartValues`? Just having this be `values` in this context is not clear what these values are being used for. Ideally I'd have like the property within Helm Chart to be named something other than `values` as well but it's too late for that now.